special-pages/playwright.config.js

```diff
@@ -96,7 +96,8 @@ export default defineConfig({
     retries: process.env.CI ? 2 : 0,
     /* Opt out of parallel tests on CI. */
     workers: process.env.CI ? 1 : undefined,
-    reporter: process.env.CI ? 'github' : [['html', { open: 'never' }]],
+    /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+    reporter: 'html',
```

Comment:
for UI stuff, the github reporter is useless. This gives us the trace+video if anything fails for easier debugging